docs/setup/jetson.md

```diff
@@ -113,6 +113,11 @@ The recommended way to run ROS2 on Jetson is to use pre-built Docker container i
 3. Set the default network interface by setting Cyclone DDS configuration.
 
         export CYCLONEDDS_URI='<CycloneDDS><Domain><General><NetworkInterfaceAddress>l4tbr0</NetworkInterfaceAddress></General></Domain></CycloneDDS>'
+    !!! attention
+        **If you are using CycloneDDS (Galactic default) and want the Jetson to talk to the robot over USB and a laptop via Wi-Fi, you will need to take extra steps to use multiple interfaces.**
+        **You will need to export a path on the Jetson to an xml config file that registers those interfaces in the CYCLONEDDS_URI.**
```

Comment:
I find this sentence a little bit confusing.
Maybe rephrase as:
"You will need to create on the Jetson a CycloneDDS XML configuration file and set the CYCLONEDDS_URI environment variable to its path"